phs.ttl

```diff
@@ -190,6 +219,23 @@
                       skos:prefLabel "Diagnostic Method Type"@en .
 
 
+###  http://linked.data.gov.au/def/phs#Bycatch
+:Bycatch rdf:type owl:Class ;
+      rdfs:subClassOf :Stratum ,
```

Comment:
the subclass of bycatch to stratum may not be correct. review why this was done. it will only be a subclass of taxon